packages/alfa-time/src/timeout.ts

```diff
@@ -0,0 +1,35 @@
+import { Time } from "./time";
+
+export class Timeout {
+  public static of(timeout: number, time: Time = Time.now()): Timeout {
```

Comment:
The "issue" with type aliases in TypeScript is that they're just, well, aliases. That means that more often than not, they will simply be expanded in intellisense and error messages. So, while the declaration site may benefit a little from introducing a type alias, call sites won't. I'm therefore not too inclined to introduce such aliases.

The other "issue" is of course the fact that time isn't necessarily measured in milliseconds; neither `Time` nor `Timeout` make any assumptions about the unit. Yes, if the default `Date.now()` is relied upon, time will be measured in milliseconds. But, it could also be measured in ticks, seconds, or hours; it's up to the caller to decide.